server/tests/unit/common/test_api.py

```diff
@@ -510,6 +510,7 @@ def setUpClass(cls, mock_get):
         cls.config = AppConfig()
         cls.config.update_server_config(
             data_locator__api_base=cls.data_locator_api_base,
+            app__web_base_url="https://cellxgene.staging.single-cell.czi.technology.com",
```

Comment:
Are we sure we want to hardcode this here (even if it's a test)? I think this should be available somewhere else. If not, I would at least try to parametrize the environment.

Reply:
No different from line 509 above?